grammars/coffeescript.cson

```diff
@@ -139,7 +139,7 @@
 
   }
   {
-    'begin': '/(?=(?![ /*+?])([^\\\\]|\\\\.)*?/[igmy]{0,4}(?![a-zA-Z0-9]))'
+    'begin': '/(?=(?![/*+?])([^\\\\]|\\\\.)*?/[igmy]{0,4}(?![a-zA-Z0-9]))'
```

Comment:
Now that we're on this, consider further improve it to

```
/(?![/*+?])(?=([^\\\\]|\\\\.)*?/[gimuy]*)
```

and with the end pattern 😉